htdocs/core/lib/loan.lib.php

```diff
@@ -82,3 +83,50 @@ function loan_prepare_head($object)
 
     return $head;
 }
+
+/**
+ * Calculate remaining loan mensuality and interests
+ *
+ * @param   int     $mens		Value of this mensuality (interests include, set 0 if we don't paid interests for this mensuality)
+ * @param   float   $capital    Remaining capital for this mensuality
+ * @param   float   $rate		Loan rate
+ * @param   int     $echance	Actual loan term
+ * @param   int   	$nbterm  	Total number of term for this loan
+ * @return  array				Array with remaining capital, interest, and mensuality for each remaining terms
+ */
+function loan_calcmens($mens, $capital, $rate, $echance, $nbterm)
```

Comment:
Even if this is not cas for old method name, for all new method introduced, we must follow the camelCase naming convention.
Can you rename into

loanCalcMonthlyPayment

Reply:
I do that today.

Reply:
Done.